### ISSUE TITLE
Drop DinD; use normal k8s workload for ember builds

### ISSUE THREAD

**Issue by Datamance** [lookit/lookit-api#566]
**TL;DR**: There is really no good reason to be using DinD. We already have containerized workloads. "Turtles all the way down" is not a design pattern. Furthermore, privileged containers are [a _bad_ idea](https://www.trendmicro.com/en_us/research/19/l/why-running-a-privileged-container-in-docker-is-a-bad-idea.html) from a security standpoint.

By taking the build process "out of the nest", we will save space, time, and cloud platform expenses.

**Narrative**
As a Developer, I want the ember build process to be as simple as possible, and to employ no more resources than are needed. The existence of superfluous nested containers is not only confusing and unnecessarily convoluted, but potentially insecure.

**Acceptance Criteria**

- [ ] Builder workloads no longer use the DinD container
- [ ] New tests for build process!
